Models/DTO/Administrator/PharmaciesReportDTO.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace API.Models.DTO.Administrator
+{
+    public class PharmaciesReportDTO
+    {
+        [JsonProperty("allamount")]
+        public decimal AllAmount { get; set; }
```

Comment:
It is unclear what `allamount` defines. I suggest clarifying this,

Reply:
pharmaciesamount